tensorflow/lite/experimental/micro/kernels/cmsis-nn/fully_connected.cc

```diff
@@ -111,7 +111,23 @@ TfLiteStatus EvalQuantizedInt8(TfLiteContext* context, TfLiteNode* node,
           data->output_activation_min, data->output_activation_max, buf),
       ARM_MATH_SUCCESS);
 #else
-#error ARM_MATH_DSP and ARM_MATH_LOOPUNROLL must be set
+
```

Comment:
Actually, it would be good for the user to be notified that there's no optimization for this target. You can add a directive for that. For example:
#pragma message("CMSIS-NN optimization not available for this target. Using ref kernel.")

Reply:
done!